bokeh/plotting_helpers.py

```diff
@@ -318,6 +318,71 @@ def _tool_from_string(name):
 
         raise ValueError("unexpected tool name '%s', %s tools are %s" % (name, text, nice_join(matches)))
 
+
+def _add_tools_to_plot(plot, tools, tools_to_remove=None):
+    """ Adds tools to the plot object skipping those specified in
+    tools_to_remove.
+
+    Args:
+        plot (Plot): instance of a plot object
+        tools (list(Tool)|str): list of tool types or string listing the
```

Comment:
in fact, you can accept a list(Tool), tuple(Tool), list(str), tuple(str) and the string list all the needed tools... maybe is better to add this info in some way...